### DISCUSSION TITLE
Interval column should return in ISO format (or have a way to do so)

### DISCUSSION THREAD

**Discussion by icopp** [supabase/supabase#15348] [Feature Requests]
Currently any `interval` column returns in `postgres` format (e.g. a string like `3 days 17 hours 33 seconds`). It would be much more useful if this had a way to return an ISO-formatted string, which can be more easily parsed by various tools and libraries.

**Comment by steve-chavez** [comment]
You mean ISO 8601? i.e. doing:

```sql
SET intervalstyle = 'iso_8601';

select '3 days 17 hours 33 seconds'::interval;
  interval
------------
 P3DT17H33S
(1 row)
```

That does look supported on https://momentjs.com/docs/#/durations/.
